### DISCUSSION TITLE
Can I pass a file from an file input to another one?

### DISCUSSION THREAD

**Discussion by MarianaAthayde** [react-hook-form/react-hook-form#3480] [Q&A]
I have a form that uses some fields to insert values on a fieldArray to show it on a material table component. 

Some inputs are file inputs so when I try to append it values to fieldArray I take the following error:

"Uncaught (in promise) DOMException: Failed to set the 'value' property on 'HTMLInputElement': This input element accepts a filename, which may only be programmatically set to the empty string."

Is there a way to do it or is ti impossible? Could someone help me, please?

Anyway tys a lot. 

CodeSandBox with the problem: https://codesandbox.io/s/file-with-fieldarraytemplate-u8bwl

**Comment by bluebill1049** [comment]
no, you can not for security reason.
